tests/features/event.feature

```diff
@@ -0,0 +1,260 @@
+  Scenario: As an editor when I create an Event node, the required fields are correctly marked when not filled in.
+    Given I am logged in as a user with the "create oe_event content, access content, edit own oe_event content, view published skos concept entities, create av_portal_photo media" permission
+    # Create a "Media AV portal photo".
+    And I visit "the AV Portal photo creation page"
+    And I fill in "Media AV Portal Photo" with "https://audiovisual.ec.europa.eu/en/photo/P-038924~2F00-15"
+    And I press "Save"
+
+    # Create a "Event" content.
+    When I visit "the Event creation page"
+    And I fill in "Title" with "My Event item"
+    And I select "As planned" from "Status"
+    And I select "Info days" from "Type"
+    And I fill in "Subject" with "financing"
+    And I press "Description"
+    And I fill in "Summary for description" with "Description summary text"
+    And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
+    And I fill in "Responsible department" with "Audit Board of the European Communities"
+    When I press "Save"
+    Then I should see the following error messages:
+      | error messages                                                                  |
+      | You have to fill in at least one of fields Internal organiser or Organiser name |
+    # Make sure that errors related to the Organiser fields is fixed.
+    When I press "Organiser"
+    And I check "Organiser is internal"
+    And I fill in "Internal organiser" with "Audit Board of the European Communities"
+    And I press "Save"
+    Then I should see the following success messages:
+      | success messages                      |
+      | Event My Event item has been created. |
+
+    # Make sure that validation of the Online fields group work as expected.
+    When I click "Edit"
+    And I press "Online"
+    And I select "Facebook" from "Online type"
+    And I press "Save"
+    Then I should see the following error messages:
+      | error messages                       |
+      | Online time start field is required. |
+      | Online link field is required.       |
+    # Make sure that errors related to the Organiser fields is fixed.
+    When I press "Online"
+    And I fill in "Online time start" with the date "02/22/2019"
+    And I fill in "Online time start" with the time "02:22:00AM"
+    And I fill in "Online time end" with the date "02/22/2019"
+    And I fill in "Online time end" with the time "02:22:00PM"
+    And I fill in "Online description" with "Online description text"
+    And I fill in "URL" with "http://ec.europa.eu/2" in the "online link"
+    And I fill in "Link text" with "Online link" in the "online link"
+    And I press "Save"
+    Then I should see the following success messages:
+      | success messages                      |
+      | Event My Event item has been updated. |
+
+    # Make sure that validation of the Description fields group work as expected.
```

Comment:
`# Make sure that validation of the Description fields group works as expected.`